src/adminSite.ts

```diff
@@ -80,9 +80,19 @@ class DefaultAdminSite {
 
       if (!column) {
         // Manytomany
+
         const relation = metadata.findRelationWithPropertyPath(property)
         const repo = this.connection.getRepository(relation.type)
-        cleanedValues[property] = await repo.findByIds(value)
+
+        let values
+        // filter out empty string because that is sent as a default value to avoid not sending anything
+        if (Array.isArray(value)) {
+          values = await repo.findByIds(value.filter(it => it != ''))
```

Comment:
If we have an array, shouldn't we just remove the first value? it should be the `hidden` input, and it avoids relying on the its value. Also, it would avoid bugs if one of the options has the actual value `''`!